src/pages/billing/Project.js

```diff
@@ -27,9 +51,51 @@ const ProjectDetail = ({ project, project: { projectName, creationStatus }, bill
   const [billingAccountName, setBillingAccountName] = useState(null)
   const [showBillingModal, setShowBillingModal] = useState(false)
   const [selectedBilling, setSelectedBilling] = useState()
+  const [tab, setTab] = useState(query.tab || 'workspaces')
 
   const signal = Utils.useCancellation()
 
+  const tabToTable = {
+    workspaces: div({ style: { flexGrow: 1, width: '100%' } }, [
+      _.map(([i, workspace]) => h(WorkspaceCard, { workspace, key: i }), Utils.toIndexPairs(_.map(response => response.workspace, _.filter(response => response.workspace.namespace === projectName, workspaces))))
+    ]),
+    users: h(Fragment, [
+      h(NewUserCard, {
+        onClick: () => setAddingUser(true)
+      }),
+      div({ style: { flexGrow: 1 } },
+        _.map(([i, member]) => {
+          return h(MemberCard, {
+            key: i,
```

Comment:
Key here shouldn't be the index again, I'd encourage you to use `'member'` if it's a string.